synapseclient/evaluation.py

```diff
@@ -1,30 +1,28 @@
 """
-***********
-Evaluations
-***********
+# Evaluations
 
-An evaluation_ object represents a collection of Synapse Entities that will be processed in a particular way.  This
+An evaluation_ object represents a collection of Synapse Entities that will be processed in a particular way. This
 could mean scoring Entries in a challenge or executing a processing pipeline.
 
-Imports::
+## Imports:
 
     from synapseclient import Evaluation, Submission, SubmissionStatus
 
-Evaluations can be retrieved by ID::
+## Evaluations can be retrieved by ID:
 
     evaluation = syn.getEvaluation(1901877)
 
-Like entities, evaluations are access controlled via ACLs. The :py:func:`synapseclient.Synapse.getPermissions` and
-:py:func:`synapseclient.Synapse.setPermissions` methods work for evaluations:
+Like entities, evaluations are access controlled via ACLs. The [synapseclient.Synapse.getPermissions][] and
+[synapseclient.Synapse.setPermissions][] methods work for evaluations:
 
     access = syn.getPermissions(evaluation, user_id)
 
-The :py:func:`synapseclient.Synapse.submit` method returns a Submission_ object::
+The [synapseclient.Synapse.submit][] method returns a Submission_ object:
```

Comment:
For `Submission_` we can update that as well to look something like:
`[Submission][synapseclient.evaluation.Submission]`

Which should add an autoreference to the class definition further on the page.